go.mod

```diff
@@ -28,15 +28,15 @@ require (
 	github.com/jacobsa/go-serial v0.0.0-20180131005756-15cf729a72d4
 	github.com/lestrrat-go/jwx v1.2.25
 	github.com/mitchellh/mapstructure v1.5.0
-	github.com/pion/ice/v2 v2.3.34
+	github.com/pion/ice/v2 v2.3.37
```

Comment:
Still Pion?

Reply:
yeah I missed a spot in viamrobotics/ice. Fixed that and propagated through webrtc -> this pr